### PR TITLE
Added Tutorial for LOJ-1037 Agent 47 (en)

### PR DESCRIPTION
**Pull request by Talha-Taki002**

### CODE REVIEW

File: 1037/en.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                        dp[mask] = min(dp[mask], dp[mask ^ (1<<i)] + cost[j][i]);
+                    }
+                }
+            }
+        }
+        cout << "Case " << ts << ": " << dp[(1<<n)-1] << '\n';
+    }
+    
+    return 0;
+}
```
Comment (rebornplusplus) [inline 1037/en.md:87]: Code-block ending symbols (` ``` `) missing.

**Review by rebornplusplus — CHANGES_REQUESTED** [review]

File: 1037/en.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+We have to compute the minimum number of shots to kill all targets using **KM .45 Tactical (USP)** and weapons from the arsenals of already killed targets.
+
+## Prerequisite
+
+Bitmask Dynamic Programming
+
+Great source for learning bitmask DP for the first time: https://www.hackerearth.com/practice/algorithms/dynamic-programming/bit-masking/tutorial/
+
+## Solution
+For each mask we brute force over all distinct pairs (**i**, **j**) **contained in the mask** where considering **i's** being the last to kill, we try weapons from **j's** arsenals to kill **it** and take the minimum result overall. This will ensure the optimum result for that particular mask. Doing that for all masks in increasing order, and our result will be the mask that contains every target.
```
[the remark this answers is not in the document]
Comment (Talha-Taki002) [reply 1037/en.md]: fixed it

**Review by rebornplusplus — APPROVED** [review]
Looks good enough for now. Feel free to work on it later if you want to formally define your functions and transitions. Good work. :tada:

**Comment by Talha-Taki002** [comment]
> Looks good enough for now. Feel free to work on it later if you want to formally define your functions and transitions. Good work. 🎉

Noted! Thanks for approving my work, man

**Pull request merged**